Firmware/cardreader.cpp

```diff
@@ -578,9 +577,9 @@ void CardReader::getStatus(bool arg_P)
         SERIAL_PROTOCOL('/');
         SERIAL_PROTOCOLLN(filesize);
         uint16_t time = ( _millis() - starttime ) / 60000U;
-        SERIAL_PROTOCOL(itostr2(time/60));
+        SERIAL_PROTOCOL(time / 60);
         SERIAL_PROTOCOL(':');
-        SERIAL_PROTOCOLLN(itostr2(time%60));
```

Comment:
Not sure why we were limiting the output here to 2 characters. Doesn't make sense to me so I just removed the limitation.